.github/workflows/conformance.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         /usr/local/bin/kubectl apply -f ./config/crd/experimental/policy.networking.k8s.io_adminnetworkpolicies.yaml
         /usr/local/bin/kubectl apply -f ./config/crd/experimental/policy.networking.k8s.io_baselineadminnetworkpolicies.yaml
         # install kube-network-policies
-        /usr/local/bin/kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kube-network-policies/v0.2.0/install-anp.yaml
+        /usr/local/bin/kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kube-network-policies/main/install-anp.yaml
```

Comment:
kind in 0.24.0 already has kube-network-policies included, so you can remove this line

Reply:
my bad, let's wait for https://github.com/kubernetes-sigs/kube-network-policies/pull/102 , it should be able to handle two implementations in parallel, just this one has the feature flags enabled for network policies

Reply:
this one https://github.com/kubernetes-sigs/kube-network-policies/pull/101